tools/modules/EnsEMBL/Web/ConfigPacker.pm

```diff
@@ -154,4 +156,21 @@ sub _vep_config_warning {
   warn $message." - thrown by ".__FILE__."\n";
 }
 
+sub _configure_assembly_converter_files {
+  my $self = shift;
+  my $species = $self->species;
+  my $chain_file_dir = File::Spec->catfile($SiteDefs::ENSEMBL_CHAIN_FILE_DIR, $species);
+  if (-d $chain_file_dir) {
+    my @files = ();
+    # Get all the available chain files to add entry to corresponding species
+    find(sub {
+      my $file = $_;
+      my $dir  = [split /\//, $File::Find::dir]->[-1];
```

Comment:
what is this line doing? is $dir being used?

Reply:
you are right. This is not being used. Removed now. thanks